README.md

```diff
@@ -35,26 +35,13 @@ advanced settings.
 Testing
 -------
 
-Make sure you have a python virtual environment with `xblock-sdk` version `0.1.5` or higher:
+1. Mimic the installation instructions for firefox & geckodriver from .travis.yml
```

Comment:
If you are going to edit testing instructions, please to so in a explicit way in the documentation. There's no need to start from xenial and install all the requirements that Travis needs in order to run the tests.